app/views/posts/show.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Here is a post POST for user X </h1>
+<div class="m-4" style="display:flex; flex-direction:column; align-items:stretch; row-gap:2vh">
+  <div>
+    <div class="rounded-lg p-8 items-center border">
+      <div class="flex justify-between">
+        <h3 class="font-bold">Post #1 by <%= @user.name %></h3>
+        <h5 class="text-red-900 font-bold mt-4 text-right"> comments: <%= @post.comments_counter %>, likes: <%= @post.likes_counter %></h5>
+      </div>
+      <p class=""><%= @post.text %> </p>
+    </div>
+ <% if @post.recent_comments.any? %>
+      <% @post.recent_comments.each do |comment| %>
+       <div class="border mt-4">
+          <p class="font-bold" style="padding: 2rem"> <%= comment.author.name %>: Comment <%= comment.id %> <p>
+        </div>
+      <% end %>

```

Comment:
- [ ] Please currently the user can see the comment id and the comment writer name , but he can't see the text of the comment please add it so the user can see the comment info .